ExamenProeveUnity/Assets/Scripts/Runtime/Player/PlayerGrabIndicationHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using Utilities.MethodExtensions;
+
+namespace Runtime.Player
+{
+    public class PlayerGrabIndicationHandler : MonoBehaviour
+    {
+        [SerializeField] private List<PlayerGrab> _playerGrabs = new();
+        
+        public UnityEvent onPlayerInRange;
+        public UnityEvent onPlayerOutRange;
+
+        private void Update()
+        {
+            if (_playerGrabs.IsEmpty()) return;
```

Comment:
maybe awake? en dan script uitzetten als het empty is (kleine game dus boeit niet veel maar toch)